server/server.js

```diff
@@ -129,7 +129,55 @@ app.patch("/:itemId", async function (req, res, next) {
   }
 });
 
-app.get("/:userId", async (req, res, next) => {});
+app.get("/:userId/wishlist", async (req, res, next) => {
+  const id = req.params.userId;
+  try {
+    const wishlist = await Users.find({ _id: id }, { wishlist: 1 });
+    res.status(200).send(wishlist);
+  } catch (err) {
+    console.log(err);
+    res.status(400).send();
+  }
+});
+
+app.patch("/:userId/wishlist", async (req, res, next) => {
+  const id = req.params.userId;
+  const item = req.body;
+  try {
+    const newItem = await Users.findOneAndUpdate(
+      { _id: id },
+      {
+        $push: {
+          wishlist: item
+        }
+      });
```

Comment:
will need to include the parameter {returnNewDocument:true} in you findOneAndUpdate call or it will return the unmodified version to you